packages/ui/src/ui/containers/ManageTokens/ManageTokensModalContent/ManageTokensModalContent.tsx

```diff
@@ -285,7 +285,7 @@ const AuthenticationTokensSection: FC<{
                             width: 110,
                             className: block('table-cell'),
                             render: ({value}) => {
-                                return `${String(value)}...`;
+                                return value ? `${String(value)}...` : 'unknown';
```

Comment:
We prefer `format.NO_VALUE` for missing values. Is it important to see the word `unknown` here?